CHANGELOG.md

```diff
@@ -716,3 +716,4 @@ Breaking changes:
 - [git] added `git sync` and `git publish` actions
 - [navigator] added the ability to toggle hidden files in the navigator
 - `jdt.ls` download on postinstall
+- `workspace.supportMultiRootWorkspace` is enabled by default
```

Comment:
This line should not be under the version v.0.3.11, but under the next release on top of the file (v0.10.1 or v0.11)